### PR TITLE
JSCB SM array EQR120 upgrade

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Replacing EQR120s at JCSB with new ones running upgraded firmware version 2.3.2
- LC 21 46541 to 46571
- LC 22 46542 to 46608
- LC 23 46543 to 46619

### CODE REVIEW

**Review by CBurton90 — DISMISSED** [review]

File: install/recorders.csv [hunk 1]
```diff
@@ -728,6 +730,7 @@ Canterbury Seismic Instruments,EQR120,EQR120,46614,MJCB,2E,23,,0,-92.5,2023-05-2
 Canterbury Seismic Instruments,EQR120,EQR120,46615,MJCB,2F,23,,0,-98,2023-05-25T03:15:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46617,HOCS,20,0,,0,0,2023-07-10T23:10:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46618,MENS,20,0,,0,0,2023-06-30T03:08:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQr120,EQR120,46619,JSCB,23,252,,0,-2.5,2024-02-29T02:57:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/recorders.csv:733]: Missed the capital R here 😆

**Review by mnaguit — COMMENTED** [review]
Thanks @Thomas-Benson , I'll approve once the naughty **r** is fixed 😄

**Comment by Thomas-Benson** [comment]
Typo fixed 😕

**Review by mnaguit — APPROVED** [review]
Thank you @Thomas-Benson 👍

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
To add, the difference in closing-opening times per LC is tricky, but you did it well @Thomas-Benson 🥇 
Probably easier to make it the same for all LCs next time especially for arrays with like 16 recorders 😆

**Pull request merged**